packages/content-element/package.json

```diff
@@ -0,0 +1,8 @@
+{
+  "name": "@tailor-cms/tce-custom-element",
+  "version": "0.0.1",
+  "description": "Content element main package for edit and display templates",
```

Comment:
Consider omitting the latter part (having something like `Tailor content element package`)